packages/edit-site/src/components/global-styles/block-preview-panel.js

```diff
@@ -53,9 +54,12 @@ const BlockPreviewPanel = ( { name, variation = '' } ) => {
 						{
 							css: `
 								body{
-									min-height:${ previewHeight }px;
-									display:flex;align-items:center;justify-content:center;
+									padding: 24px;
```

Comment:
We could add a comment here about keeping this CSS in sync with the lines above.